zeroconf.py

```diff
@@ -1705,9 +1706,41 @@ def find(cls, zc=None, timeout=5, interfaces=InterfaceChoice.All):
         return tuple(sorted(listener.found_services))
 
 
+if "ANDROID_ARGUMENT" in os.environ:
+    from jnius import autoclass
```

Comment:
Maybe mention in the README that you need to install pyjnius if using on Android